include/vrb/ModelLoaderAndroid.h

```diff
@@ -24,6 +24,7 @@ class ModelLoaderAndroid : public LoaderThread {
   void InitializeGL();
   void ShutdownGL();
   void LoadModel(const std::string& aModelName, GroupPtr aTargetNode);
+  void LoadModel(vrb::LoadTask aLoadTask, GroupPtr aTargetNode);
```

Comment:
Why not just use `RunLoadTask()` directly? Also why are the parameters swapped from the `RunLoadTask()` function?

Reply:
Only to keep consistency with other `LoadControllerModel`methods that call `LoadModel` but no major reason. Same as the parameters order, just because the other `LoadModel` methods have the target node as second parameter.

Thanks for the review btw ;)

Reply:
Just in case, this is part of this: https://github.com/MozillaReality/FirefoxReality/pull/3940